### PR TITLE
Made headers on cases and judges tables bold

### PR DESCRIPTION
**Pull request by c-melchor**
The headers on the Cases and Judges tables needed to be bold in order to give users a clear indication of what kind of data each column of the tables holds.

### CODE REVIEW

File: src/components/pages/CaseTable/CaseTable.js [hunk 1]
```diff
@@ -608,6 +608,7 @@ export default function CaseTable(props) {
           {drawerContent()}
         </Drawer>
       </div>
+      {/* <div className={classes.dataGridContainer}> */}
```
Comment (JusticeMatthew) [inline src/components/pages/CaseTable/CaseTable.js:611]: Please remove commented code :)

File: src/components/pages/CaseTable/CaseTable.js [hunk 2]
```diff
@@ -620,6 +621,7 @@ export default function CaseTable(props) {
         disableColumnMenu={true}
         components={{ Toolbar: Toolbar }}
       />
+      {/* </div> */}
```
Comment (JusticeMatthew) [inline src/components/pages/CaseTable/CaseTable.js:624]: Please remove commented code :)

**Review by JusticeMatthew — COMMENTED** [review]
Excellent work! Looks great, very interesting how the renderHeader works 🤔

File: src/components/pages/CaseTable/CaseTable.js (hunk 1 above)
Comment (c-melchor) [reply src/components/pages/CaseTable/CaseTable.js]: Ah, thank you! Done.

**Review by JusticeMatthew — APPROVED** [review]
Looks good!

**Review by cmirza — APPROVED** [review]
Looks good to me

**Review by reesharper — APPROVED** [review]
Looks great! Love the use of the strong tag here.

**Pull request merged**